plugins/woocommerce-admin/client/hooks/usePreventLeavingPage.ts

```diff
@@ -20,7 +26,9 @@ export default function usePreventLeavingPage(
 			__( 'Changes you made may not be saved.', 'woocommerce' ),
 		[ message ]
 	);
-	const { navigator } = useContext( NavigationContext );
+	const nav = useContext( NavigationContext );
+	const { navigator } = nav;
```

Comment:
Is there a reason to change these lines? I'm not sure if it's worth adding the extra line.